### ISSUE TITLE
Close button in the filter section is not working

### ISSUE THREAD

**Issue by sdgun** [topcoderinc/va-online-memorial-frontend#9]
## Target URL

https://frontend.app.cloud.gov/search

## Steps to Reproduce

1. Open the URL https://frontend.app.cloud.gov/
2. Login as a valid User
3. Click Search
4. Select 'Branch of service' as US NAVY
5. Click 'Update Results' button
6. Click close filter button

## Actual Result(s)

- Close button in the filter section is not working

## Expected Result(s)

- Must clsoe the filter

## Frequency

- 5/5

## Environment Details

- **Device(s):** Laptop
- **Operating System:** Windows 7 64bit
- **Browser(s):** Chrome 67.0.3396.87 (Official Build) (64-bit)

## Image(s)/Video(s)/Console Log/Crash Log

![4](https://user-images.githubusercontent.com/4592712/41741048-93acac86-755f-11e8-99b8-d980dece6202.jpg)

**Comment by ramillim** [comment]
Is there a design/wireframe that shows how this should look like when you re-open it? Does it collapse/expand or is it close and it's gone for good until you reload the page?

**Comment by ramillim** [comment]
Hi @sdgun . I just wanted to check if you had a design spec for how this should behave.

**Comment by sdgun** [comment]
Actually I don’t have but @dustinWeaver might have these documents

**Comment by dustinWeaver** [comment]
There's nothing this detailed.  However, upon looking this up, I think we should just remove the close button.  I can't imagine why a person would want to close the filter list seeing as it doesn't expand the screen or anything like that.

**Comment by ramillim** [comment]
That's true. All that happens is the footer comes up. I wasn't sure if there was another view that could possibly use this where this was behaving more like a modal, but nothing that I've seen.

**Comment by ramillim** [comment]
The only thing I was thinking was if we wanted to collapse it so that the results table takes up the full width.

**Comment by sdgun** [comment]
Hi @ramillim @dustinWeaver , I have checked this site on a tablet view. This close button is used to CLOSE the filter section once it is expanded. So I think remove is not the suitable thing to do. Hide it from desktop viewport is the best thing to do I guess

**Comment by dustinWeaver** [comment]
@ramillim, I like @sdgun's observation above.   I didn't test on a tablet, so she has a good point.
